src/i18n/en.json

```diff
@@ -35,6 +35,7 @@
     "button.next": "Next",
     "button.save": "Save",
     "button.modify": "Modify",
+    "button.action": "New Action",
```

Comment:
This is not a general button name, so it should go as a message descriptor inside the `ColonyHomeActions` component